### PR TITLE
Implement saving and loading of cookies

### PR DESCRIPTION
**Pull request by basnijholt**

### CODE REVIEW

**Comment by basnijholt** [comment]
@sudoguy I am not really familiar with how the Mock class works inside the test. Do you know what I would have to do to set the cookie?

File: instabot/api/api.py [hunk 1]
```diff
@@ -45,31 +47,37 @@ def set_user(self, username, password):
         self.password = password
         self.uuid = self.generate_UUID(uuid_type=True)
 
-    def login(self, username=None, password=None, force=False, proxy=None):
+    def login(self, username=None, password=None, force=False, proxy=None,
+              use_cookie=False, cookie_fname='cookie.txt'):
```
Comment (sudoguy) [inline instabot/api/api.py]: Why is ```use_cookie``` set False by default?
Maybe we should set this parameter to True?

**Review by sudoguy — DISMISSED** [review]

File: instabot/api/api.py (hunk 1 above)
Comment (basnijholt) [reply instabot/api/api.py]: Yes, that is probably true, I'll change it.

**Review by sudoguy — APPROVED** [review]
Awesome!

**Pull request merged**